android/libraries/rib-android-compose/build.gradle

```diff
@@ -18,6 +18,11 @@ apply plugin: 'com.android.library'
 apply plugin: 'org.jetbrains.kotlin.android'
 apply plugin: "com.vanniktech.maven.publish"
 
+kotlin {
```

Comment:
We should probably apply this by default to all build targets (with the couple exceptions until we convert/fix them), so this isn't managed in ~20 spots.

Reply:
Agreed with the general statement of centralizing Kotlin module configurations. I can try to move those two to the project-level build.gradle.

With that said, the reason I did not do it on our project-level `build.gradle` is IMO we do config centralization poorly, in a non-Gradle-idiomatic way. We should probably be using a Gradle conventions plugin instead of those hard-to-read (to me) configurations at project-level, either built from `buildSrc` or `includedBuilds`, then we could just apply the plugin like
```
plugins {
 id("ribs-kotlin-convention-plugin")
}
```

This [seems to be](https://docs.gradle.org/current/samples/sample_convention_plugins.html) the Gradle-way for applying common configurations between different modules.

Perhaps we could open an issue to improve on that.

Reply:
Agreed. Let's punt this, and create an issue for the plugin, as well as moving the remaining java modules over and enabling explicitApi on the outlier.